task/fossa-scanner/0.1/test/run.yaml

```diff
@@ -0,0 +1,55 @@
+---
+apiVersion: tekton.dev/v1beta1
+kind: TaskRun
+metadata:
+  name: fossa-scan-springboot
```

Comment:
CI is unable to run this particular test as `run.yaml` is present inside `test/` instead of `/tests`. Is this expected ?

Reply:
got it, will change it